pom.xml

```diff
@@ -138,6 +137,16 @@
         <enabled>true</enabled>
       </releases>
     </repository>
+    <repository>
+      <id>snapshot-repo</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
why it is snapshot repo? we cannot take jars from snapshot repo.  Do you have any plan to keep this jar in maven central repo?